### PR TITLE
Improve org dashboard

### PR DESCRIPTION
**Pull request by axelsson**
For the org dashboard:
* Added top 3 climate action to org dashboard
* Fixed some styling (padding, font sizes and stuff)
* Added images above numbers for org climate actions and tonnes offset
* Realized I had named the org-dashboard folder with - instead of _ so i changed that, which makes the history a bit messy, sorry about that.

For climate actions:
I added climate actions on the "take action" page (http://localhost:3000/take-action) below "monthly subscription".
It links to the dashboard using an anchor so the user sees the climate actions right away.

I'll ask Cissi to look through the copy and images afterwards but if you see anything that's clearly off, please do tell.

### CODE REVIEW

**Comment by axelsson** [comment]
[Add climate actions in https://www.goclimate.com/take-action](https://trello.com/c/5NZ9p0b7/805-add-climate-actions-in-https-wwwgoclimatecom-take-action)

**Comment by axelsson** [comment]
[Additional content for org dashboard](https://trello.com/c/3yHhRPUl/828-additional-content-for-org-dashboard)

**Review by kallenilver — APPROVED** [review]
I think it looks good! Good job! 🥳

File: config/locales/views/organization_dashboard.en.yml [hunk 1]
```diff
@@ -3,10 +3,12 @@ en:
     organization_dashboard:
       greeting: Hello, climate friend!
       climate_actions: Climate actions employees of %{orgName} have done
-      tonnes_avoided: Tonnes CO2 avoided through climate projects
-      employees: We are now <b>%{orgEmployeesCount} employees of %{orgName} </b>who calculated our footprint. For every action you do, %{orgName} finances a climate project with 10 EUR.
+      tonnes_avoided: CO2 avoided through climate projects
+      employees: You are now <b>%{orgEmployeesCount} employees of %{orgName} </b>who calculated your footprint. For every action you do, %{orgName} finances a climate project with 10 EUR.
```
Comment (nanditagoclimate) [inline config/locales/views/organization_dashboard.en.yml:7]: 💯

**Review by nanditagoclimate — APPROVED** [review]

File: app/controllers/organization_controller.rb [hunk 2]
```diff
@@ -19,10 +19,24 @@ def show
...
+      .joins('JOIN user_climate_actions ON user_climate_actions.climate_action_id = climate_actions.id
+        JOIN climate_action_categories ON climate_action_categories.id = climate_actions.climate_action_category_id')
+      .where('user_climate_actions.user_id IN (?)', user_ids)
+      .select('climate_actions.*,
+        climate_action_categories.name as category_name,
+        count(user_climate_actions.climate_action_id) as total')
+      .order('COUNT(user_climate_actions.climate_action_id) DESC')
+      .group('climate_actions.id, climate_action_categories.name')
+      .limit(3)
+  end
```
Comment (nanditagoclimate) [inline app/controllers/organization_controller.rb:38]: 💯

**Pull request merged**